modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -5,25 +5,30 @@ The search service maintains a cluster-wide set of index definitions and metadat
-The search service redistributes the index partitions across the available search service nodes for a balanced partition-node assignment during a rebalance operation. The newly assigned index partitions are built afresh over DCP feed on the new nodes. And once the new partitions are built up to the current / latest sequence numbers, they are promoted to take the live traffic, and the older partitions are deleted from the system. 
+The search service redistributes the index partitions across the available search service nodes for a balanced partition-node assignment during a rebalance operation. The newly assigned index partitions are built afresh over DCP feed on the new nodes. Once the new partitions are built up to the current / latest sequence numbers, the partitions are promoted to handle live traffic, and the older partitions are deleted from the system. 
+
+FTS Rebalance operation involving FTS nodes does not wait for indexes' replicas partitions to be built entirely. While doing the FTS rebalance, the users need to know that this is the expected behavior.
 
 == Impacts on the live traffic updates
 
-The live traffic is never functionally affected. Nevertheless, we can't entirely rule out the performance impacts of a concurrent rebalance on a live cluster.
+The live traffic is never functionally affected. However, we cannot entirely rule out the performance impacts of a background rebalance operation on the live cluster traffic. 
+If the user has mutations happening on their data service nodes, the new data will continue to be ingested into the FTS indexes despite the rebalance/failover operation.
 
 == Tips for faster rebalance operations
 
-As the rebalance is a resource-consuming cluster management operation, It is always recommended to perform rebalance during off-peak hours. Search service moves or builds the index partitions one at a time per node during the rebalance operations, which could significantly increase the overall time taken for the rebalance operation.
-
+As the rebalance is a resource-consuming cluster management operation, it is always recommended to perform rebalance during off-peak hours.Search service moves or builds the index partitions one at a time per node during the rebalance operations, which could significantly increase the overall time taken for the rebalance operation.
+ 
 There is a configurable option [maxConcurrentPartitionMovesPerNode] to bring the additional concurrency to how we move/build partitions during a rebalance operation. 
-Suppose we override this parameter (maxConcurrentPartitionMovesPerNode to N) as a runtime cluster option. In that case, we could concurrently build that many partitions in parallel per node at a time, and the rebalance ought to complete faster.
+ 
+If we override this parameter(maxConcurrentPartitionMovesPerNode to N) as a runtime cluster option, then we could concurrently build that many partitions in parallel per node at a time, and the rebalance ought to complete faster.
```

Comment:
By overriding the maxConcurrentPartitionMovesPerNode parameter using the runtime cluster option, we can control the concurrency of partition moves, enabling faster rebalance completion times.

Knowing what value to configure this parameter depends on the particular use case and available network bandwidth. Adequate testing must be done in a pre-production environment prior to deploying production clusters with this setting.

Reply:
@sreekanth-cb - I've added some more content here re. testing in a pre-production environment and slightly made the sentence easier to read. Can you please take a look?

Reply:
👍

Reply:
Updated